<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesMedia</class>
 <widget class="QWidget" name="PreferencesMedia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">

   <item>
    <widget class="QLabel" name="label_help_thumbnails1">
     <property name="toolTip">
      <string extracomment="Help info 1"/>
     </property>
     <property name="text">
      <string>UPDATE THUMBNAILS</string>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QLabel" name="label_help_thumbnails2">
     <property name="toolTip">
      <string extracomment="Help info 1"/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Photographs or videos are sometimes edited after being uploaded and linked to Subsurface. If a photo is therefore more recent than its original thumbnail, checking the checkbox below will allow creating a new thumbnail after the media has been edited.</string>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QCheckBox" name="auto_recalculate_thumbnails">
     <property name="text">
      <string>Recalculate thumbnail if older than media file</string>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QLabel" name="label_help_media_1">
     <property name="toolTip">
      <string extracomment="Help info 1"/>
     </property>
     <property name="text">
      <string> </string>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QLabel" name="label_help_video_1">
     <property name="toolTip">
      <string extracomment="Help info 1"/>
     </property>
     <property name="text">
      <string>VIDEOS</string>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="title">
      <string>Video thumbnails</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>5</number>
      </property>

      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_help_video_2">
        <property name="toolTip">
         <string extracomment="Help info 2"/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>One can view video files through thumbnail(s) in the Media tab, created using the external program FFmpeg which needs to be installed on your machine. Check the checkbox below to allow this. Then, in the text box below, specify the path to FFmpeg. In Linux, a typical path is:  /usr/bin/ffmpeg. If FFmpeg is installed in the system area of your computer, just type ffmpeg in the text box below. For more info see Appendix F of the User Manual.</string>
        </property>
       </widget>
      </item>

      <item row="1" column="0">
       <widget class="QCheckBox" name="extractVideoThumbnails">
        <property name="text">
         <string>Extract video thumbnails</string>
        </property>
       </widget>
      </item>

      <item row="2" column="0">
       <widget class="QLabel" name="ffmpegExectuableLabel">
        <property name="text">
         <string>Path to ffmpeg:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="ffmpegExecutable"/>
        </item>
        <item>
         <widget class="QToolButton" name="ffmpegFile">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>

      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_help_video_3">
        <property name="toolTip">
         <string extracomment="Help info 3"/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>One can specify te relative position within the video from where the thumbnail is retrieved. Use the slider below do do that.</string>
        </property>
       </widget>
      </item>

      <item row="4" column="0">
       <widget class="QLabel" name="videoThumbnailPositionLabel">
        <property name="text">
         <string>Extract at position</string>
        </property>
       </widget>
      </item>

      <item row="4" column="1">
       <widget class="QSlider" name="videoThumbnailPosition">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>

     </layout>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="title">
      <string>Dive data as subtitles</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>5</number>
      </property>

      <item row="0" column="0">
       <widget class="QLabel" name="subtitlesFormatStringLabel">
        <property name="text">
         <string>Subtitle format string:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLineEdit" name="subtitlesFormatString"/>
        </item>
       </layout>
      </item>
      
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_help_video_4">
        <property name="toolTip">
         <string extracomment="Help info 4"/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Available tags:</string>
        </property>
       </widget>
      </item>
      
      <item row="2" column="0">
       <widget class="QLabel" name="subtitlesTimeLabel">
        <property name="text">
         <string>Dive time: [time]</string>
        </property>
       </widget>
      </item>

      <item row="3" column="0">
       <widget class="QLabel" name="subtitlesDepthLabel">
        <property name="text">
         <string>Depth: [depth]</string>
        </property>
       </widget>
      </item>
      
      <item row="4" column="0">
       <widget class="QLabel" name="subtitlesTemperatureLabel">
        <property name="text">
         <string>Temperature: [temperature]</string>
        </property>
       </widget>
      </item>

      <item row="5" column="0">
       <widget class="QLabel" name="subtitlesCeilingLabel">
        <property name="text">
         <string>Ceiling: [ceiling]</string>
        </property>
       </widget>
      </item>

      <item row="6" column="0">
       <widget class="QLabel" name="subtitlesNdlLabel">
        <property name="text">
         <string>NDL: [ndl]</string>
        </property>
       </widget>
      </item>

      <item row="7" column="0">
       <widget class="QLabel" name="subtitlesTtsLabel">
        <property name="text">
         <string>TTS: [tts]</string>
        </property>
       </widget>
      </item>

      <item row="8" column="0">
       <widget class="QLabel" name="subtitlesRbtLabel">
        <property name="text">
         <string>RBT: [rbt]</string>
        </property>
       </widget>
      </item>

      <item row="9" column="0">
       <widget class="QLabel" name="subtitlesStoptimeLabel">
        <property name="text">
         <string>Stop time: [stoptime]</string>
        </property>
       </widget>
      </item>

      <item row="10" column="0">
       <widget class="QLabel" name="subtitlesStopdepthLabel">
        <property name="text">
         <string>Stop depth: [stopdepth]</string>
        </property>
       </widget>
      </item>

      <item row="11" column="0">
       <widget class="QLabel" name="subtitlesCnsLabel">
        <property name="text">
         <string>CNS: [cns]</string>
        </property>
       </widget>
      </item>

      <item row="12" column="0">
       <widget class="QLabel" name="subtitlesSacLabel">
        <property name="text">
         <string>SAC: [sac]</string>
        </property>
       </widget>
      </item>

      <item row="13" column="0">
       <widget class="QLabel" name="subtitlesPO2Label">
        <property name="text">
         <string>pO₂: [p_o2]</string>
        </property>
       </widget>
      </item>

      <item row="14" column="0">
       <widget class="QLabel" name="subtitlesPN2Label">
        <property name="text">
         <string>pN₂: [p_n2]</string>
        </property>
       </widget>
      </item>

      <item row="15" column="0">
       <widget class="QLabel" name="subtitlesPHeLabel">
        <property name="text">
         <string>pHe: [p_he]</string>
        </property>
       </widget>
      </item>

      <item row="16" column="0">
       <widget class="QLabel" name="subtitlesO2pressureLabel">
        <property name="text">
         <string>O₂ pressure (rebreather): [o2_pressure]</string>
        </property>
       </widget>
      </item>

      <item row="17" column="0">
       <widget class="QLabel" name="subtitlesO2setpointLabel">
        <property name="text">
         <string>O₂ setpoint: [o2_setpoint]</string>
        </property>
       </widget>
      </item>

      <item row="18" column="0">
       <widget class="QLabel" name="subtitlesScr_oc_p02Label">
        <property name="text">
          <string>SCR ΔpO₂: [scr_oc_po2]</string>
        </property>
       </widget>
      </item>

      <item row="2" column="1">
       <widget class="QLabel" name="subtitlesModLabel">
        <property name="text">
         <string>MOD: [mod]</string>
        </property>
       </widget>
      </item>

      <item row="3" column="1">
       <widget class="QLabel" name="subtitlesEadLabel">
        <property name="text">
         <string>EAD: [ead]</string>
        </property>
       </widget>
      </item>

      <item row="4" column="1">
       <widget class="QLabel" name="subtitlesEndLabel">
        <property name="text">
         <string>END: [end]</string>
        </property>
       </widget>
      </item>

      <item row="5" column="1">
       <widget class="QLabel" name="subtitlesEaddLabel">
        <property name="text">
         <string>EADD: [eadd]</string>
        </property>
       </widget>
      </item>

      <item row="6" column="1">
       <widget class="QLabel" name="subtitlesSpeedLabel">
        <property name="text">
         <string>Vertical speed: [speed]</string>
        </property>
       </widget>
      </item>

      <item row="7" column="1">
       <widget class="QLabel" name="subtitlesIn_deco_calcLabel">
        <property name="text">
         <string>In deco (calculated): [in_deco]</string>
        </property>
       </widget>
      </item>

      <item row="8" column="1">
       <widget class="QLabel" name="subtitlesNdlCalcLabel">
        <property name="text">
         <string>NDL (calculated): [ndl_calc]</string>
        </property>
       </widget>
      </item>

      <item row="9" column="1">
       <widget class="QLabel" name="subtitlesTts_calcLabel">
        <property name="text">
         <string>TTS (calculated): [tts_calc]</string>
        </property>
       </widget>
      </item>

      <item row="10" column="1">
       <widget class="QLabel" name="subtitlesStoptime_calcLabel">
        <property name="text">
         <string>Stop time (calculated): [stoptime_calc]</string>
        </property>
       </widget>
      </item>

      <item row="11" column="1">
       <widget class="QLabel" name="subtitlesStopdepth_calcLabel">
        <property name="text">
         <string>Stop depth (calculated): [stopdepth_calc]</string>
        </property>
       </widget>
      </item>

      <item row="12" column="1">
       <widget class="QLabel" name="subtitlesHeartrateLabel">
        <property name="text">
         <string>Heartrate: [heartrate]</string>
        </property>
       </widget>
      </item>

      <item row="13" column="1">
       <widget class="QLabel" name="subtitlesBearingLabel">
        <property name="text">
         <string>Bearing: [bearing]</string>
        </property>
       </widget>
      </item>

      <item row="14" column="1">
       <widget class="QLabel" name="subtitlesSurface_gfLabel">
        <property name="text">
         <string>Surface GF: [surface_gf]</string>
        </property>
       </widget>
      </item>

      <item row="15" column="1">
       <widget class="QLabel" name="subtitlesCurrent_gfLabel">
        <property name="text">
         <string>Current GF: [current_gf]</string>
        </property>
       </widget>
      </item>

      <item row="16" column="1">
       <widget class="QLabel" name="subtitlesDensityLabel">
        <property name="text">
         <string>Density: [density]</string>
        </property>
       </widget>
      </item>

      <item row="17" column="1">
       <widget class="QLabel" name="subtitlesIcd_warningLabel">
        <property name="text">
         <string>ICD Warning: [icd_warning]</string>
        </property>
       </widget>
      </item>

     </layout>
    </widget>
   </item>

   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>

  </layout>
 </widget>

 <resources/>
 <connections>
 </connections>
</ui>
